### ISSUE TITLE
Error adding user

### ISSUE THREAD

**Issue by Akasurde** [vmware/govmomi#821]
Adding new user raises following error

  ```
  managed object not found: UserDirectory:UserDirectory
  ```

**Comment by anfernee** [comment]
@Akasurde Same thing here. Can you clarify what command did you run, and what version of vcsim did you use?

**Comment by Akasurde** [comment]
@anfernee Thanks for response.

I am facing this issue while using vcsim with Ansible module called `vmware_local_user_manager.py`

`vmware_local_user_manager.py` uses an API 

```
content.userDirectory.RetrieveUserGroups(...)
```
This is where vcsim fails.

You can reproduce this using following Ansible Playbook against latest vcsim code.

```yaml
---
- name: Add User
  hosts: localhost
  tasks:
    - name: Add user
      vmware_local_user_manager:
        validate_certs: false
        hostname: 0.0.0.0
        username: user
        password: pass
        local_user_name: dev_0001
        local_user_password: dev123
        local_user_description: Sample Development user
        state: present
```

**Comment by dougm** [comment]
Fixed in #855
